package.json

```diff
@@ -32,7 +32,7 @@
     "docs:toc": "doctoc README.md",
     "lint": "vjsstandard",
     "start": "npm-run-all -p watch start:*",
-    "start:serve": "babel-node scripts/server.js",
+    "start:serve": "babel-node scripts/server.js --presets env",
```

Comment:
also, using a `.babelrc` file would mean we can just have preset set in one place instead of sprinkled liberally everywhere :)
However, this is probably fine too.

Reply:
See https://github.com/videojs/mpd-parser/blob/master/.babelrc for an example from another one of our projects

Reply:
I think moving to babel-preset-env is a great idea, we just need to make sure we tell it to target the browsers we care about.

Reply:
I've moved the generic babel config into a .babelrc